### DISCUSSION TITLE
Python study sources

### DISCUSSION THREAD

**Discussion by Rodrigo-dev-bit** [1101-GS4719-C/Python-Community-GS4719-C#758] [Ideas]
跟大家分享一下我最近複習python的資料:
1. [Corey Schafer's YouTube tutorials](https://www.youtube.com/playlist?list=PL-osiE80TeTskrapNbzXhwoFUiLCjGgY7) - **影片**
2. [Python For Everybody](http://do1.dr-chuck.com/pythonlearn/EN_us/pythonlearn.pdf) - **pdf**
3. 還有我們老師的教材 - **eeclass**

上面1.跟2.是全英文，要是有看不懂或聽不懂的地方，可以在下面留言，我~們~會盡量幫忙翻譯。

**Comment by Chiway1229** [comment]
謝謝你提供的教材 我之前也有透過codecademy學習一些python的語法

**Reply by Rodrigo-dev-bit** [reply]
codecademy我之前也有用過，蠻不錯的。

**Comment by williamyen0623** [comment]
感謝整理分享！

**Comment by danwan1027** [comment]
感謝分享！

**Comment by Zoe06060606** [comment]
我之前有上過python for everybody 那個教授的線上課程 他教的真的很好 上課也很有趣👍

**Comment by a45320614** [comment]
感謝分享~ 不過我下載Python For Everybody的檔案時，瀏覽器提醒有潛在的安全風險，如果可以直接提供來源連結就更好了!

**Reply by Rodrigo-dev-bit** [reply]
Python for everybody 的original source 本來是從 Dr. Charles R. Severance(教授）from University of Michigan 所寫的，是從他的課拿到的- https://www.coursera.org/specializations/python

**Comment by Pietra1226** [comment]
謝謝分享，很好用
